src/farming/custom-plants.ts

```diff
@@ -0,0 +1,358 @@
+
+interface Plant {
+  seeds: CustomItem<{}>;
+  minAge: number;
+  maxAge: number;
+  drops: ItemStack[];
+  /* From 0 to 1. 1 = always grows, 0 = never grows*/
+  growth?: number;
+}
+
+const PLANT_BLOCK = Material.TWISTING_VINES;
+
+/**
+ * Plant 1
+ */
+const ExamplePlant1Seeds = new CustomItem({
+  id: 1,
+  modelId: 1,
+  type: VkItem.SEED,
+});
+const ExamplePlant1: Plant = {
+  seeds: ExamplePlant1Seeds,
+  minAge: 0,
+  maxAge: 2,
+  drops: [ExamplePlant1Seeds.create(), new ItemStack(Material.WHEAT, 2)],
+  growth: 0.2,
+};
+
+/**
+ * Plant 2
+ */
+const ExamplePlant2Seeds = new CustomItem({
+  id: 2,
+  modelId: 2,
+  type: VkItem.SEED,
+});
+const ExamplePlant2: Plant = {
+  seeds: ExamplePlant2Seeds,
+  minAge: 3,
+  maxAge: 5,
+  drops: [ExamplePlant2Seeds.create(), new ItemStack(Material.APPLE, 5)],
+};
+
+/**
+ * Plant 3
+ */
+const ExamplePlant3Seeds = new CustomItem({
+  id: 3,
+  modelId: 3,
+  type: VkItem.SEED,
+});
+const ExamplePlant3: Plant = {
+  seeds: ExamplePlant3Seeds,
+  minAge: 6,
+  maxAge: 8,
+  drops: [ExamplePlant3Seeds.create(), new ItemStack(Material.CARROT, 3)],
+};
+
+/**
+ * Plant 4
+ */
+const ExamplePlant4Seeds = new CustomItem({
+  id: 4,
+  modelId: 4,
+  type: VkItem.SEED,
+});
+const ExamplePlant4: Plant = {
+  seeds: ExamplePlant4Seeds,
+  minAge: 9,
+  maxAge: 11,
+  drops: [ExamplePlant4Seeds.create(), new ItemStack(Material.GRASS)],
+};
+
+/**
+ * Plant 5
+ */
+const ExamplePlant5Seeds = new CustomItem({
+  id: 5,
+  modelId: 5,
+  type: VkItem.SEED,
+});
+const ExamplePlant5: Plant = {
+  seeds: ExamplePlant5Seeds,
+  minAge: 12,
+  maxAge: 14,
+  drops: [ExamplePlant5Seeds.create(), new ItemStack(Material.GRASS)],
+};
+
+/**
+ * Plant 6
+ */
+const ExamplePlant6Seeds = new CustomItem({
+  id: 6,
+  modelId: 6,
+  type: VkItem.SEED,
+});
+const ExamplePlant6: Plant = {
+  seeds: ExamplePlant6Seeds,
+  minAge: 15,
+  maxAge: 17,
+  drops: [ExamplePlant6Seeds.create(), new ItemStack(Material.GRASS)],
+};
+
+/**
+ * Plant 7
+ */
+const ExamplePlant7Seeds = new CustomItem({
+  id: 7,
+  modelId: 7,
+  type: VkItem.SEED,
+});
+const ExamplePlant7: Plant = {
+  seeds: ExamplePlant7Seeds,
+  minAge: 18,
+  maxAge: 20,
+  drops: [ExamplePlant7Seeds.create(), new ItemStack(Material.GRASS)],
+};
+
+/**
+ * Plant 8
+ */
+const ExamplePlant8Seeds = new CustomItem({
+  id: 8,
+  modelId: 8,
+  type: VkItem.SEED,
+});
+const ExamplePlant8: Plant = {
+  seeds: ExamplePlant8Seeds,
+  minAge: 21,
+  maxAge: 25,
+  drops: [ExamplePlant8Seeds.create(), new ItemStack(Material.GRASS)],
+};
+
+/**
+ * Plant 9
+ */
+const ExamplePlant9Seeds = new CustomItem({
+  id: 9,
+  modelId: 9,
+  type: VkItem.SEED,
+});
+const ExamplePlant9: Plant = {
+  seeds: ExamplePlant9Seeds,
+  minAge: 24,
+  maxAge: 25,
+  drops: [ExamplePlant9Seeds.create(), new ItemStack(Material.GRASS)],
+};
+
+/**
+ * Maps the age of the twisting vines to the corresponding plant
+ */
+const AGE_TO_PLANT = new Map<
```

Comment:
Computing this based on content of plant objects would be nice (but certainly not required).